src/main/java/model/Order.java

```diff
@@ -0,0 +1,55 @@
+package model;
+
+import java.util.Objects;
+
+public class Order {
+
+    private String id;
+
+    private Product product;
```

Comment:
Pro Bestellung kann nur _ein_ einziges Produkt bestellt werden - da wäre eBay vlt weniger glücklich drüber (mit einer `List<Product>` könnte man pro Bestellung _mehrere_ Produkte bestellen - wäre vlt. noch nice für die User), _aber_ funktioniert ;)

Pluspunkt: Du hast eine sogenannte _Assoziation_ umgesetzt, d.h. benutzt ein Javaobjekt in einem anderen - top!